src/main.rs

```diff
@@ -64,6 +73,7 @@ fn main() {
 
     let my_path = &args[1];
     let mut nemo_to_find = "";
+    let mut output_file = "";
```

Comment:
Don't use empty string as a placeholder: Option<&str> is the better way

Reply:
Fixed